specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.EventGrid/EventGrid.tsp

```diff
@@ -55,13 +55,11 @@ model SubscriptionValidationEventData {
   /** The validation code sent by Azure Event Grid to validate an event subscription.
    * To complete the validation handshake, the subscriber must either respond with this validation code as part of the validation response,
    * or perform a GET request on the validationUrl (available starting version 2018-05-01-preview). */
```

Comment:
(apologies, can't leave a comment on unchanged code)

A few of the comments in here make reference to IDs that aren't valid (they have `SystemEvents` in them, but it's not in their online docs):

* "Microsoft.EventGrid.SystemEvents.MQTTClientCreatedOrUpdated"
* "Microsoft.EventGrid.SystemEvents.MQTTClientDeleted"
* "Microsoft.EventGrid.SystemEvents.MQTTClientSessionConnected",
* "Microsoft.EventGrid.SystemEvents.MQTTClientSessionDisconnected"
* "Microsoft.EventGrid.SystemEvents.SubscriptionDeletedEvent",
* "Microsoft.EventGrid.SystemEvents.SubscriptionValidationEvent"

The MQTT ones are here, for example:
https://learn.microsoft.com/en-us/azure/event-grid/event-schema-event-grid-namespace?tabs=cloud-event-schema

I just noticed this when I regen'd and they came up differently than last time.

Reply:
I ended up adding these to my PR to this PR. So SOON.